server/src/sessions/service/sessions.service.ts

```diff
@@ -319,6 +323,12 @@ export class SessionsService {
       sessionId: session.id,
     })
 
-    return this.getSession(session.id)
+    if (!('playerInSession' in session) || session.playerInSession.length < 1) {
+      await this.sessionRepository.delete(session.id)
+      await this.gamesService.remove(session.gameId)
```

Comment:
Maybe add a cascade to the relation of game and session so if the game is deleted, the session is also deleted.

Reply:
Cascades added